skimage/_shared/fft.py

```diff
@@ -12,14 +12,6 @@
 except ImportError:
     import numpy.fft
     fftmodule = numpy.fft
-
-    try:
-        from scipy.fftpack import next_fast_len
-    except ImportError:
-        # next_fast_len was implemented in scipy 0.18
-        # In case it cannot be imported, we use the id function
-        def next_fast_len(size):
-            """Dummy next_fast_len that returns size unmodified."""
-            return size
+    from scipy.fftpack import next_fast_len
```

Comment:
We could start importing from `scipy.fft` instead once the minimum SciPy is 1.4, but it should not be strictly necessary. 

Imports from `scipy.fftpack` should continue to work despite the FFTPack fortran code being removed from Scipy 1.4. (If you are not aware, SciPy 1.4 will have a new `scipy.fft` module. Given the amount of legacy code using the `fftpack` module, we do not intend to remove it. The functions from `scipy.fftpack` now just call functions from `scipy.fft` under the hood.)

Reply:
This task is already noted here: https://github.com/scikit-image/scikit-image/blob/master/TODO.txt#L64 :)

Reply:
by you I guess ;)